app/scripts/services/StlCourtsDatepickerInputValidation.svc.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+angular.module('yourStlCourts').factory('StlCourtsDatePickerInputValidation', function () {
```

Comment:
Yes, thank you.  I had initially thought I would make use of a service and coded the shell, but then never needed it and forgot to delete it.  I've removed it now.